driver-core/src/test/java/com/datastax/driver/core/CaseSensitivityTest.java

```diff
@@ -36,12 +37,21 @@ private void assertNotExists(CCMBridge.CCMCluster c, String name) {
 
     @Test(groups = "short")
     public void testCaseInsensitiveKeyspace() throws Throwable {
-        CCMBridge.CCMCluster c = CCMBridge.buildCluster(1, Cluster.builder());
+        CCMBridge.CCMCluster c = CCMBridge.buildCluster(1, Cluster.builder()
+            .withQueryOptions(new QueryOptions()
+                .setRefreshNodeIntervalMillis(0)
+                .setRefreshNodeListIntervalMillis(0)
+                .setRefreshSchemaIntervalMillis(0)
+            )
+        );
         Session s = c.session;
         try {
             String ksName = "MyKeyspace";
             s.execute(String.format(CREATE_KEYSPACE_SIMPLE_FORMAT, ksName, 1));
 
+            // give some time for schema events to be debounced
+            Thread.sleep(500);
```

Comment:
Has to add lots of these to make the driver pause until the events are debounced :( - even setting intervals to zero is not enough.